src/components/Calculator.js

```diff
@@ -1,46 +1,93 @@
 import React from 'react';
+import calculate from '../logic/calculate';
 
 class Calculator extends React.PureComponent {
+  constructor(props) {
+    super(props);
+    this.state = {
+      total: null,
+      next: null,
+      operation: null,
+    };
+  }
+
+  clickReceived = (event) => {
+    const answer = calculate(this.state, event.target.innerText);
+    this.setState((prevState) => ({
+      ...prevState,
+      ...answer,
+    }));
+  };
```

Comment:
- [ ] Just as the previous reviewer highlighted, for this task, you are expected to have a functional component with hooks. Kindly make your component a functional component t and make use of the `useState` hook provided by react. Kindly refactor your code and make use of the useState hook as requested in the project requirements. 

```javascript
// ./src/components/Calculator.js

import React, { useState } from 'react';
import calculate from ../logic; // Import math logic

function Calculator() {
    // This below can be your state with the of useState hook
    const [total, setTotal] = useState(0);
    return {
      <div>
       // Calculator app 
      </div>
    };
}
```

Reply:
I was thinking you guys were actually requesting me to implement the next project which is the Refactor the Calculator component (components/Calculator.js) from a class-based component to a functional component with hooks. I need some clarification on that because am working on math-magicians: event and not the refactor with hooks.

Reply:
In the review guide and the project requirement for this milestone, you're asked to use the `useState` hook and I believe the useState hook is only used in functional components and not class-based components. Kindly check the project requirements again, you'll see that it's stated that you use useState hook and the example code I gave in the review comment, tells that, that's how you're expected to implement the component and how to use the useState hook. 👍🏽

Reply:
Thanks for the response